au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/BalanceKnob.qml

```diff
@@ -0,0 +1,62 @@
+/*
+* Audacity: A Digital Audio Editor
+*/
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import Muse.Ui 1.0
+import Muse.UiComponents 1.0
+
+KnobControl {
+    id: root
+    isBalanceKnob: true
+
+    StyledPopupView {
+        id: popup
+        placement: PopupView.Above
+        openPolicies: PopupView.NoActivateFocus
+        padding: 8
+        margins: 8
+        contentWidth: contentRect.width
+        contentHeight: contentRect.height
+        FontMetrics {
+            id: fontMetrics
+            font: content.font
+        }
+        property rect contentRect: fontMetrics.boundingRect("Pan: 100R")
+
+        Item {
+            anchors.fill: parent
+            id: content
+            Text {
+                anchors.fill: parent
+                text: "Pan:"
+            }
+
+            Text {
+                anchors.right: parent.right
+                text: {
+                    let value = Math.round(root.value);
+                    let direction = value < 0 ? 'L' : value > 0 ? 'R' : '';
+                    return `${Math.abs(value)}${direction}`;
+                }
+            }
+        }
+    }
+
+    Timer {
+        id: popupTimer
+        interval: 200
```

Comment:
This value should be taken from the settings.
It needs to be improved, add something like a delay tooltip to the Theme property. Take it from the uiconfiguration (need add method)

Reply:
Done